packages/repository-json-schema/src/filter-json-schema.ts

```diff
@@ -50,43 +55,61 @@ export function getScopeFilterJsonSchemaFor(
  * a generic json schema allowing any "where" condition.
  *
  * @param modelCtor - The model constructor to build the filter schema for.
+ * @param options - Options to build the filter schema.
  */
 export function getFilterJsonSchemaFor(
   modelCtor: typeof Model,
   options: FilterSchemaOptions = {},
 ): JsonSchema {
-  const schema: JsonSchema = {
-    ...(options.setTitle !== false && {
-      title: `${modelCtor.modelName}.Filter`,
-    }),
-    properties: {
-      where: getWhereJsonSchemaFor(modelCtor, options),
+  let excluded: string[];
+  if (typeof options.exclude === 'string') {
+    excluded = [options.exclude];
+  } else {
+    excluded = options.exclude ?? [];
+  }
+  if (typeof excluded === 'string') excluded = [excluded];
```

Comment:
I think this last check is redundant, because `excluded` can be a string only when `options.exclude` was a string, which was already handled.

```suggestion
  let excluded: string[];
  if (typeof options.exclude === 'string') {
    excluded = [options.exclude];
  } else {
    excluded = options.exclude ?? [];
  }
```